CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 
+## [ 0.7.2 ] - [ 2024-11-20 ]
+
+### Added
+
+- Integrate with libqasm 0.6.7 release.
+- Add `reset` instruction.
```

Comment:
Does this mean that QX not only accepts the newer version of libQASM, but also that the `reset` instruction is implemented in QX?

Reply:
That's right. It was implemented a long time ago. Last QX simulator release is from May. `reset` was added to QX simulator between May and September.